app/models/cart.rb

```diff
@@ -1,10 +1,11 @@
 class Cart
   include ActionView::Helpers::NumberHelper
 
-  attr_reader :contents
+  attr_reader :contents, :quantity
 
   def initialize(initial_contents)
     @contents = initial_contents || {}
+    @quantity = contents.values.sum
```

Comment:
We actually need that attr_reader for the @contents variable though @julsfelic. I'll be sure to check out the quantity one right now and update you on that!